lamindb_setup/core/_hub_core.py

```diff
@@ -98,13 +97,11 @@ def delete_instance(instance_identifier: str) -> None:
         # we need to make sure the default 0-byte object in the storage location has
         # not been deleted to avoid permission errors from leveraging s3fs on an
         # empty subdirectory
-        path = create_path(
-            instance_account["storage"]["root"], settings.user.access_token
-        )
-        path.fs.touch(str(path / TOUCH_FILE_PATH))
+        root = instance_account["storage"]["root"]
+        mark_storage_root(root)
 
         # gate storage and instance deletion on empty storage location
-        check_s3_storage_location_empty(path)
+        check_s3_storage_location_empty(root)
```

Comment:
This reads global credentials. If you want to get hosted credentials here, you need to use `create_path`.

Reply:
This is why this fails for the hosted instance in the test, it doesn't have hosted credentials.